portfolio/src/main/webapp/style.css

```diff
@@ -74,31 +102,49 @@
   align-items: center;
   font-size: 20px;
 }
+
 .gallery:hover {
   transform: scale(1.1);
   opacity: 0.7;
 }
+
 .gallery img {
-  width:100%;
+  width: 100%;
```

Comment:
Does the 'width: 100%' have any impact? If you remove this line will the result be the same?